### DISCUSSION TITLE
v2.4.0 --  when client connet, the server response: Error during WebSocket handshake: Unexpected response code: 400

### DISCUSSION THREAD

**Discussion by Jason-oo** [socketio/socket.io#3739] [Q&A] [answered]
In Docker run `npm install`, container install socket.io@2.4.0, 
as my client can't connect to my server, before this I use socket.io@2.3.0, and there is no problem, 
is there any wrong with v2.4.0? 
Has anyone been in the same situation? plz

**Comment by darrachequesne** [comment, marked as answer]
Hi! Are you using CORS in your application? Did you check the release notes? https://github.com/socketio/socket.io/releases/tag/2.4.0

**Reply by Jason-oo** [reply]
thx, I have resolved this
